.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
+OPENAI_API_KEY=
```

Comment:
Ensure that these environment variables are actually being used in your application code. If they are not utilized, it may confuse future maintainers.